modules/kubernetes/pages/quickstart-with-eks.adoc

```diff
@@ -2,12 +2,16 @@
 :description: A quickstart guide for running TigerGraph in Kubernetes in EKS.
 :cloud-code: eks
 :cloud-caps: EKS
+:sectnums:
 
 This is a quickstart guide for deploying TigerGraph in Kubernetes on AWS's Elastic Kubernetes Service (EKS).
 
 * <<Single-server deployment>>
 * <<Cluster deployment>>
 
+WARNING: Known issue:
+The GUI service, which is the service for GraphStudio and AdminPortal UI, does not currently support high availability in an EKS cluster.
+
```

Comment:
What does this mean for the user? Maybe say more directly that GraphStudio and Admin Portal (with space) will not be available if TigerGraph is run on an EKS cluster